controllers/ContainerController.php

```diff
@@ -70,7 +70,9 @@ public function actionView()
         } elseif ($page->type == Container::TYPE_MARKDOWN) {
             return $this->render('markdown', array('page' => $page, 'md' => $page->page_content));
         } elseif ($page->type == Container::TYPE_TEMPLATE) {
-            return $this->viewTemplatePage($page);
+	        return $this->viewTemplatePage($page);
+        } elseif ($page->type == Container::TYPE_HTML) {
+	        return $this->render('html', ['page' => $page, 'html' => $page->page_content, 'title' => $page->title]);
         } elseif ($page->type == Container::TYPE_PHP) {
             return $this->render('php', [
```

Comment:
the indentation is a bit off in the return statements (compared to e.g. line 71)

Reply:
fixed